web/tests/stream_list.test.js

```diff
@@ -265,61 +265,63 @@ function add_row(sub) {
     stream_list.stream_sidebar.set_row(sub.stream_id, row);
 }
 
+const develSub = {
+    name: "devel",
+    stream_id: 1000,
+    color: "blue",
+    pin_to_top: true,
+    subscribed: true,
+};
+
+const RomeSub = {
+    name: "Rome",
+    stream_id: 2000,
+    color: "blue",
+    pin_to_top: true,
+    subscribed: true,
+};
+
+const testSub = {
+    name: "test",
+    stream_id: 3000,
+    color: "blue",
+    pin_to_top: true,
+    subscribed: true,
+};
+
+const announceSub = {
+    name: "announce",
+    stream_id: 4000,
+    color: "green",
+    pin_to_top: false,
+    subscribed: true,
+};
+
+const DenmarkSub = {
+    name: "Denmark",
+    stream_id: 5000,
+    color: "green",
+    pin_to_top: false,
+    subscribed: true,
+};
+
+const carSub = {
+    name: "cars",
+    stream_id: 6000,
+    color: "green",
+    pin_to_top: false,
+    subscribed: true,
+};
```

Comment:
Just FYI, we can start thinking about converting modules like this to use `web/tests/lib/test_stream.js` while spending time on them, if it's helpful.

Reply:
At this point I think it's easier to keep it as it is here already.

Reply:
Yeah agreed, just wanted to spread awareness.